docs/src/content/docs/utilities/Signals/computed-from.md

```diff
@@ -71,7 +73,8 @@ let c = computedFrom(
 			([a, b]) =>
 				// of(a + b) is supposed to be an asynchronous operation (e.g. http request)
 				of(a + b).pipe(delay(1000)) // delay the emission of the combined value by 1 second for demonstration purposes
-		)
+		),
+		{ initialValue: 42 } // change the initial value of the resulting signal
```

Comment:
We don't need this here as signals have an initial value by default, so it's not the case. 
```suggestion
```